src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -152,5 +152,10 @@
         <service id="translation.extractor" class="%translation.extractor.class%"/>
 
         <service id="translation.writer" class="%translation.writer.class%"/>
+
+        <service id="translation.warmer" class="Symfony\Bundle\FrameworkBundle\CacheWarmer\TranslationsCacheWarmer" public="false">
+            <argument type="service" id="translator" />
```

Comment:
I'm sorry, I'm lost. I tried to understand what is this service ("translator.logging.inner") but actually when I search for this string into symfony, I just can find this service id being injected into a LoggingTranslator. I have no idea of where is this service configured or what it's supposed to do. Can you please explain it ?